gc/gc-iceberg/src/main/java/org/projectnessie/gc/iceberg/GcProcedureUtil.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.gc.iceberg;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import org.apache.iceberg.spark.procedures.BaseGcProcedure;
+import org.apache.spark.sql.catalyst.InternalRow;
+import org.apache.spark.sql.catalyst.analysis.NoSuchProcedureException;
+import org.apache.spark.sql.connector.catalog.Identifier;
+import org.apache.spark.sql.connector.catalog.TableCatalog;
+import org.apache.spark.unsafe.types.UTF8String;
+import scala.collection.JavaConverters;
+import scala.collection.Seq;
+
+final class GcProcedureUtil {
+
+  private GcProcedureUtil() {}
+
+  static final String NAMESPACE = "nessie_gc";
```

Comment:
We should document that `nessie_gc` namespace cannot be used by the Nessie clients now.

Reply:
I don't think we have that restriction. It can still be used as normal namespace.
Only restriction is that while calling the stored procedure, `nessie_gc` namespace is must. 
This restriction is already added in java doc of `NessieIcebergGcSparkCatalog` and `NessieIcebergGcSparkSessionCatalog`